src/app/shared/services/data/data.service.ts

```diff
@@ -485,24 +485,30 @@ export class DataService {
     });
   }
 
-  public downloadFile(item: RecordVO): Promise<any> {
+  public downloadFile(item: RecordVO, type?: string): Promise<any> {
     if (item.FileVOs && item.FileVOs.length) {
-      downloadOriginalFile(item);
+      downloadFile(item, type);
       return Promise.resolve();
     } else {
       return this.fetchFullItems([item]).then(() => {
-        downloadOriginalFile(item);
+        downloadFile(item, type);
       });
     }
 
-    function downloadOriginalFile(fileItem: any) {
-      const fileVO = getOriginalFile(fileItem) as any;
+    function downloadFile(fileItem: any, type?: string) {
+      const fileVO = getFile(fileItem, type) as any;
+      console.log('downloadFile', fileVO);
+      console.log('downloadFile', fileItem);
```

Comment:
It would be better to remove these console logs.

Reply:
yes you are right, i am sorry about this, i removed them

Reply:
@cecilia-donnelly can you please test now with the file with no extension?